### ISSUE TITLE
Automatic Unban setting jumps back to 5min

### ISSUE THREAD

**Issue by Yerrun** [wp-blocks/cf7-antispam#92]
I have quite some cron job times in the Automatic Unban list. When I pick one from the list (other than 5min) and save it, the Automatic Unban time is saved.

But if I change any other setting on the page and save again, the Automatic Unban time jumps back to 5min.

**Comment by erikyo** [comment]
this is a minor problem but you are right that it is not correct in operation and you should check whether the settings have changed before resetting the timer

ref. https://github.com/wp-blocks/cf7-antispam/blob/cce2c6be377e84d6840830db5ecafc77fd27ef19/admin/CF7_AntiSpam_Admin_Customizations.php#L983

**Comment by Yerrun** [comment]
Yes, it's just a small bug. But it's just annoying that you have to save twice instead of once.

**Comment by erikyo** [comment]
I just use a simple emoji enclosed in a <span> element... which is why I think it has been replaced with a website speed optimisation tool that replaces emoji with an image (even in the backend). 

That's why on one provider it happens to you and on the other it doesn't, it's a problem of the provider having to update the icon set because the ones it was based on are no longer available (or so it seem)

https://github.com/wp-blocks/cf7-antispam/blob/3e0e83eda308c8ef29b2402333a7e86b4840f9da/admin/CF7_AntiSpam_Admin_Display.php#L47
